src/Text/Pandoc/Writers/AsciiDoc.hs

```diff
@@ -61,18 +61,28 @@ data WriterState = WriterState { defListMarker    :: String
                                , bulletListLevel  :: Int
                                , intraword        :: Bool
                                , autoIds          :: Set.Set String
+                               , asciidoctorDlct  :: Bool
```

Comment:
Dlct is a bit obscure.  `asciidoctorVariant`?
or just `asciidoctor`?

Reply:
done